types/jasmine/index.d.ts

```diff
@@ -1032,36 +1032,45 @@ declare namespace jasmine {
 
 declare module "jasmine" {
     class jasmine {
-        constructor(options: any);
         jasmine: jasmine.Jasmine;
+        env: jasmine.Env;
+        reportersCount: number;
+        completionReporter: jasmine.CustomReporter;
+        reporter: jasmine.CustomReporter;
+        showingColors: boolean;
+        projectBaseDir: string;
+        specDir: string;
+        specFiles: string[];
+        helperFiles: string[];
+        requires: string[];
+        onCompleteCallbackAdded: boolean;
+        defaultReporterConfigured: boolean;
+
+        constructor(options: any);
         addMatchers(matchers: jasmine.CustomMatcherFactories): void;
         addReporter(reporter: jasmine.CustomReporter): void;
         addSpecFile(filePath: string): void;
         addSpecFiles(files: string[]): void;
+        addHelperFiles(files: string[]): void;
```

Comment:
Is this... still a real thing?
https://github.com/jasmine/jasmine/search?q=addHelperFiles

I feel like half of the things here no longer exist.

Reply:
The class is from [jasmine/jasmine-npm](https://github.com/jasmine/jasmine-npm)

https://github.com/jasmine/jasmine-npm/search?q=addHelperFiles

Reply:
Shouldn't that have its own DT package then?

Reply:
Never mind. I get it. It's an augmentation of browser jasmine to make it run on node, per https://jasmine.github.io/setup/nodejs.html.

I think you need to go with my solution below then.